enterprise_gateway/enterprisegatewayapp.py

```diff
@@ -70,6 +70,16 @@ def remote_hosts_default(self):
     def yarn_endpoint_default(self):
         return os.getenv(self.yarn_endpoint_env, self.yarn_endpoint_default_value)
 
+    # Conductor endpoint
+    conductor_endpoint_env = 'EG_CONDUCTOR_ENDPOINT'
+    conductor_endpoint_default_value = None
+    conductor_endpoint = Unicode(conductor_endpoint_default_value, config=True,
+        help="""The http url for accessing the Conductor REST API. (EG_CONDUCTOR_ENDPOINT env var)""")
+
+    @default('conductor_endpoint')
+    def conductor_endpoint_default(self):
+        return os.getenv(self.conductor_endpoint_env, self.conductor_endpoint_default_value)
+
```

Comment:
@kevin-bates should we have merge these configurations into a single RESOURCE_MANAGER_ENDPOINT type of parameter?

Reply:
@lresende. I'd rather we move the various *process proxy* configuration items to the specific classes, then perhaps those classes could have a callable *main()* method that displays information about the class including the configurable parameters.  This will take some research, so I think its best we leave the parameters separated for now and defined at more of the global scope.

The goal would be to have process proxy classes completely self-contained.  I'm just not sure these configuration items would be automatically *documented* when running `jupyter enterprisegateway --help-all` or not - although so long as there's a way for someone to get at the *help* content for the process proxy I don't view entries in `--help-all` as a requirement.

Reply:
Please see my comment in the conversation.  Let's leave this as-is for now.